core/storage/base_model/gae_models.py

```diff
@@ -193,10 +198,10 @@ def _fetch_page_sorted_by_last_updated(
             3-tuple of (results, cursor, more) as described in fetch_page() at:
             https://developers.google.com/appengine/docs/python/ndb/queryclass,
             where:
-                results: List of query results.
+                results: list(List of query results.
                 cursor: str or None. A query cursor pointing to the next batch
                     of results. If there are no more results, this will be None.
-                more: bool. Whether there are more (likely) more results after
+                more: bool. Whether there are (likely) more results after
```

Comment:
Actually, I retract this -- I now see where you got it from (the GAE docs). Instead, maybe you can write something like: "If True, there may be more results to fetch. If False, there are no further results to fetch."